about.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/about.css">
+    <script src="/components/header.js"></script>
+    <script src="/components/footer.js"></script>
+    <title>About</title>
+</head>
+<body>
+    <header-component></header-component>
+    <main class="main-section">
+        <a class="btn-hamburger" href="#">
+            <div class="main-header-hamburger">
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+            </div>
+        </a>
+        <ul class="hambuger-menu">
+            <li class="ham-menu-items">
+                <a href="#main" class="ham-menu-link">Main Section</a>
+            </li>
+            <li class="ham-menu-items">
+                <a href="#speaker" class="ham-menu-link">Feature Speakers</a>
+            </li>
+            <li class="ham-menu-items">
+                <a href="#work" class="ham-menu-link">Work Section</a>
+            </li>
+        </ul>
+        <h1 class="main-section-title">"Hello! Green World"</h1>
+        <p class="main-section-text">
+
+        </p>
+        <p class="main-section-footer">
+            please contact us per Email for and future question<br>
+            about CC Global Summit 2022~<br>
+            globalwarming@microverse.com
+        </p>
+    </main>
+    
+    <section class="common-section">
+        <h2 class="common-section-title">
+            Creative Commons<br>
+            Global Summit 2022 logo
+        </h2>
+        <p class="common-section-text">
+            
+        </p>
+        <p class="common-section-footer">
+            The logo of CC Global summit 2022 was decided<br>
+            through the logo cometition from 8 June to 7 July
+        </p>
+        <img class="common-section-logo" src="/assets/icon/logo.png" alt="logo">
+    </section>
+
+    <section class="history-section">
+        <h2 class="history-section-title">
+            See the past CC global Summits
+        </h2>
+        <div class="underline"></div>
+        <p class="history-section-text">
+            Take a look at the last two CC global Summit wich took place <br>
+            in Buenos Aires and in Warsaw
+        </p>
+        <div class="history-img-section">
+            <div>
+                <img src="/assets/images/2013.jpg" alt="2013 summit image">
+            <div class="text-center">2013 Summit</div>
+            </div>
+            <div>
+                <img src="/assets/images/2011.jpg" alt="2011 summit image">
+                <div class="text-center">2011 Summit</div>
+            </div>            
+        </div>        
+    </section>
+    <footer-component></footer-component>
```

Comment:
Kindly change the design of the footer in the desktop version to match the Behance design.
| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/157993963-07376240-9f01-4416-bf7e-6cc519c9ffd7.png) | ![image](https://user-images.githubusercontent.com/24703171/157994000-283a8d26-893f-4213-aac4-caba858d1aec.png) |